### ISSUE TITLE
Arras theme displays only 1 column when using nodes

### ISSUE THREAD

**Issue by ckcelite** [iCaspar/arras#63]
Hi, I'm using Arras theme 1.8 with Wordpress 4.9.5. A few days ago, I updated Arras to the newest version and I discovered my squares containing an exerpt and featured image of a post which used to be 3 rows of 3 columns for a total of 9 nodes now has become 1 column of 9 nodes with an unsightly blank bit on the right.

I have monkeyed with the Arras theme options (wp-admin/themes.php?page=arras-options) until I was blue in the face.

I tried disabling every plugin.

I viewed my site on Android, mobile, firefox, IE and Chrome.

Attached are two images, what it looks like now vs what it used to look like.

If anyone has any ideas how I can solve this problem, I'd be grateful!

![arras theme original working](https://user-images.githubusercontent.com/38842028/39410621-bd0ee8e6-4bc0-11e8-8ecb-435ddfeea60f.jpg)
![arras theme node single column](https://user-images.githubusercontent.com/38842028/39410622-bd200fcc-4bc0-11e8-93fc-8b6d25b76752.jpg)

**Comment by iCaspar** [comment]
Hi @ckcelite,

On the Arras Options page, under the "Layout" tab, is "Nodes per Row" set to 3 as shown below? That would be the setting that controls the x-across layout.

![screen shot 2018-04-29 at 10 12 24 pm](https://user-images.githubusercontent.com/7462998/39413589-7550f582-4bfa-11e8-921a-b6aef823fc03.png)

**Comment by iCaspar** [comment]
>  I also uninstalled the theme, installed an older version of Arras and it was still broke.

So there must be something else that's interfering, since going back to the previous Arras version didn't help. (But question is what, since you said you've already tried disabling plugins!) I went to your site to look at how display rendering, but because node display turned off (understandably) I couldn't find anything out.

**Comment by iCaspar** [comment]
Thanks for the vote of confidence.

Looking at just the public-facing front page, I can see the following:

It looks like you have a "premium addons" plugin that is redefining a style class called `clearfix`. In this screenshot you can see, circled, it's calling for those "clearfixed" nodes to clear: both, meaning: "move down until it's clear of the previous one."
![premium-addons-clearfix](https://user-images.githubusercontent.com/7462998/39499779-29631dfa-4d7e-11e8-95d5-c29dfcf810e3.png)

Now, when I disable that in the browser, it goes back to the correct 3-across display as in this screenshot:

![premium-addons-clearfix-disabled](https://user-images.githubusercontent.com/7462998/39499834-72129dd2-4d7e-11e8-850a-4c121d9620e8.png)

So the upshot is that the issue is in the premium-addons, which is a plugin I'm not familiar with.

If you can't figure out how to change the premium-addons output, you *might* be able to override their styles using the WP customizer: Go to Customizer > Additional CSS and enter an overriding rule as shown here (this may, however, cause whatever the premium-addons is doing to look odd):

![screen shot 2018-05-01 at 8 34 32 pm](https://user-images.githubusercontent.com/7462998/39499961-4a8cdad8-4d7f-11e8-8a3d-0e52c971b06f.png)

Let me know how that goes.

**Comment by ckcelite** [comment]
iCaspar, all I can say is wow, you rock! I just uninstalled the plugin and now everything's working like a charm. Thank you man : )

**So, if anyone else should have this same difficulty here's a rundown:**

There was a conflict with [Arras theme](https://arrastheme.net/) and [Premium Addons for Elementor Version 2.1.7 by Leap13](https://wordpress.org/plugins/premium-addons-for-elementor/)

My problem was solved by uninstalling the Premium Addons for Elementor plugin.
